docs/source/markdown/podman-create.1.md

```diff
@@ -817,6 +817,10 @@ container. The `OPTIONS` are a comma delimited list and can be:
 * [rw|ro]
 * [z|Z]
 * [`[r]shared`|`[r]slave`|`[r]private`]
+* [`[r]bind`]
+* [`exec`|`noexec`]
```

Comment:
Should this swap to `noexec | exec` to match the other two?

Reply:
Fixed